mrp_bom_note/README.rst

```diff
@@ -1,11 +1,18 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
     :alt: License: AGPL-3
 
-MRP - BoM notes
-===============
+==========================
+Notes in Bill of Materials
+==========================
 
 This module creates in BoM a new field to add notes
 
+Usage
+=====
+
```

Comment:
* Go to Manufacturing > ...
* Edit an existing BOM and fill in the Notes field on tab X after field Y